docs/docs/visual-testing-with-storybook.md

```diff
@@ -21,6 +21,130 @@ npx -p @storybook/cli sb init
+    // Transpile Gatsby module because Gatsby includes un-transpiled ES6 code.
+    config.module.rules[0].exclude = [/node_modules\/(?!(gatsby)\/)/]
+
+    // use installed babel-loader which is v8.0-beta (which is meant to work with @babel/core@7)
+    config.module.rules[0].use[0].loader = require.resolve("babel-loader")
+
+    // use @babel/preset-react for JSX and env (instead of staged presets)
+    config.module.rules[0].use[0].options.presets = [
+      require.resolve("@babel/preset-react"),
+      require.resolve("@babel/preset-env"),
+    ]
+
+    config.module.rules[0].use[0].options.plugins = [
+      // use @babel/plugin-proposal-class-properties for class arrow functions
+      require.resolve("@babel/plugin-proposal-class-properties"),
+      // use babel-plugin-remove-graphql-queries to remove static queries from components when rendering in storybook
+      require.resolve("babel-plugin-remove-graphql-queries"),
+    ]
+
+    // Prefer Gatsby ES6 entrypoint (module) over commonjs (main) entrypoint
+    config.resolve.mainFields = ["browser", "module", "main"];
+
+    return config;
+  },
+```
+
+The final output will look as follows:
+
+```js:title=.storybook/main.js
+module.exports = {
+  stories: ["../stories/**/*.stories.js", "../src/**/*.stories.tsx"],
+  addons: ["@storybook/addon-actions", "@storybook/addon-links"],
+  // highlight-start
+  webpackFinal: async config => {
+    // Transpile Gatsby module because Gatsby includes un-transpiled ES6 code.
+    config.module.rules[0].exclude = [/node_modules\/(?!(gatsby)\/)/]
+
+    // use installed babel-loader which is v8.0-beta (which is meant to work with @babel/core@7)
+    config.module.rules[0].use[0].loader = require.resolve("babel-loader")
+
+    // use @babel/preset-react for JSX and env (instead of staged presets)
+    config.module.rules[0].use[0].options.presets = [
+      require.resolve("@babel/preset-react"),
+      require.resolve("@babel/preset-env"),
+    ]
+
+    config.module.rules[0].use[0].options.plugins = [
+      // use @babel/plugin-proposal-class-properties for class arrow functions
+      require.resolve("@babel/plugin-proposal-class-properties"),
+      // use babel-plugin-remove-graphql-queries to remove static queries from components when rendering in storybook
+      require.resolve("babel-plugin-remove-graphql-queries"),
+    ]
+
+    // Prefer Gatsby ES6 entrypoint (module) over commonjs (main) entrypoint
+    config.resolve.mainFields = ["browser", "module", "main"]
+
+    return config
+  },
+  // highlight-end
+}
+```
+
+Next create a new file under `.storybook` called `preview.js`. This configuration file preview.js is not responsible for loading any stories. Its main purpose is to add global parameters and decorators.
+
+```js:title=.storybook/preview.js
+import { action } from "@storybook/addon-actions"
+
+// automatically import all files ending in *.stories.js
+// configure(require.context("../src", true, /\.stories\.js$/), module)
```

Comment:
@aaronsturm if you can remove these two lines as they are not accurate and no longer needed.

Reply:
Thanks for catching that, I had thought I removed those lines :(.